server/datastore/mysql/sessions.go

```diff
@@ -28,8 +30,10 @@ func (ds *Datastore) SessionByKey(ctx context.Context, key string) (*fleet.Sessi
 
 func (ds *Datastore) SessionByID(ctx context.Context, id uint) (*fleet.Session, error) {
 	sqlStatement := `
-		SELECT * FROM sessions
-		WHERE id = ?
+		SELECT s.*, u.api_only FROM sessions s
+		LEFT JOIN users u
+		ON s.user_id = u.id
+		WHERE s.id = ?
 		LIMIT 1
```

Comment:
Just curious why there's `LIMIT 1` here (Column `id` is `PRIMARY KEY`, so it will always return either zero or one result.)

Reply:
Yeah, not sure either, as you see it was already there. This could be safely removed for sure.